ui/js/controllers/jobs.js

```diff
@@ -131,6 +139,58 @@ treeherderApp.controller('ResultSetCtrl', [
 
         };
 
+        $scope.notifyWhenDone = function(revision) {
+            Notification.requestPermission().then(function(result) {
+                if(result === "granted") {
+                    if($rootScope.watchedResultsets.length === 0) {
```

Comment:
oh, sorry.  no, this just prevents re-creating the interval for the second one...  :)